.github/actions/update-docs/action.yml

```diff
@@ -44,6 +44,6 @@ runs:
           poetry run mike deploy dev --push --rebase --config-file ./docs/mkdocs.yml
         else
           new_tag=${NEW_TAG%.*}
-          poetry run mike deploy "$new_tag" latest --update-aliases --rebase --config-file ./docs/mkdocs.yml
+          poetry run mike deploy "$new_tag" --push latest --update-aliases --rebase --config-file ./docs/mkdocs.yml
           poetry run mike set-default latest --push  --rebase --config-file ./docs/mkdocs.yml
```

Comment:
is this `--push` still needed?

Reply:
Yes because withouth this push nothing is pushed to that branch.
According to the [docs](https://github.com/jimporter/mike#building-your-docs) of mike :

> In addition, you can specify where to deploy your docs via -b/--branch, -r/--remote, and --deploy-prefix, specifying the branch, remote, and directory prefix within the branch, respectively. Finally, to push your docs to a remote branch, simply add -p/--push to your command.

Reply:
I mean the second one, in the line below (`set-default`). Could that lead to the double push behavior again?